src/index.js

```diff
@@ -40,6 +40,57 @@ const _views = new WeakMap();
 
 let bypass = false;
 
+function nextOrPreviousProject (index) {
+    // Generate an array of projects by selecting children of 'project-viewer > ul[is="pv-list-tree"]'. For each 'li[is="pv-list-nested-item"]', expand it into its children.
+    const sidebar = document.getElementsByTagName('project-viewer')[0];
+    let list = sidebar.getElementsByTagName('ul')[0].children;
+
+    var output = [];
+
+    var expandProjects = function (element) {
+        if (element.getAttribute('is') === "pv-list-nested-item") {
+            let nestedList = element.getElementsByTagName('ul')[0].children;
```

Comment:
So here is where i have problems because my `element.getElementsByTagName('ul')[0]` is an empty `ul` and I actually have more elements.

I have made this little change that basically is for inside a for loop, which I am not very keen on it.

The problem is that, somewhere in the loop, it's not going through the visually alignment projects. And I can't test your version because it can't find any active project.

here it is:

```js
if (element.getAttribute('is') === "pv-list-nested-item") {
    let nestedList = element.getElementsByTagName('ul');
    for (let i = 0; i < nestedList.length; i++) {
        let nestedListChildren = nestedList[i].children;
        for (let j = 0; j < nestedListChildren.length; j++) {
            expandProjects(nestedListChildren[j]);
        }
    }
}
```